plugin/trino-tpch/src/main/java/io/trino/plugin/tpch/TpchMetadata.java

```diff
@@ -595,7 +602,7 @@ public static Type getTrinoType(TpchColumn<?> column)
             case DATE:
                 return DATE;
             case DOUBLE:
-                return DOUBLE;
+                return doubleTypeMapping.getMappedType();
```

Comment:
Seems like TpchColumnType.DOUBLE option should be renamed as well.

Reply:
Again, I don't really have any strong opinions here, so I can make that change, but the default mapping is still DOUBLE.